api/ImageData.json

```
@@ -148,6 +148,68 @@
                 "deprecated": false
               }
             }
+          },
+          "pixelFormat_option": {
+            "__compat": {
+              "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvascolortype",
+              "support": {
+                "chrome": {
+                  "version_added": "137"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
```

Firefox has a tracking bug:

```suggestion
                  "version_added": false,
                  "impl_url": "https://bugzil.la/1958830"
```